README.md

```diff
@@ -18,40 +18,99 @@ Some of these are tricky!  Don't trust your first instinct.
 
 **a)** `"1" == 1`
 
+True - Due to the == not looking for an exact similarity, but a 'truthiness'.
+Console says: True
+
 **b)** `"1" === 1`
 
+False - 3 ==='s are looking for a 100% match, and string 1 is not exactly the same as number 1.
+Console says: False
+
 **c)** `x == 'x'`
 
+True - Again, because double ='s are looking to see a close fit, rather than an exact match, and x is similar to string x. Moreover, javascript will turn x into a string in an effort to 'help out'
+Console says: Uncaught (x is not defined)
+After defining x as 1, console says: false, my assumption being that variable x is not == string x.
+
 **d)** `x == (x+'')`
 
+True - Because '' is considered an empty string and thus nothing, leaving x to be evaluated to itself.
+Console says: True (x defined as 1, otherwise says undefined)
+
 **e)** `'' == ' '`
 
+False - In this case '' is an empty string, whereas ' ' actually contains content, in this case a space.
+Console says: False
+
+
 **f)** `x = true`
 
+True - Because you are declaring x as true.
+Console says: True
+
 **g)** `var x; x == 'undefined'`
 
+False - In the first step a variable, x, has been defined.
+Console says: false (because of why I say, I'm not so sure)
+
 **h)** `'9'<'10'`
 
+True - javascript will look at these strings as numbers for the operator
+Console says: false. Because in retrospect, javascript looks at the first digit in the string, rather than the string as a whole. Essentially this turns into 9<1, which is false.
+
 **i)** `typeof x + 1 === "number"`
 
+If x has been defined earlier as a number, this will output True.
+Console says: False. After tinkering around with what typeof x + 1 is on its own (after declaring x = 1), it retrieved "number1". So the console is actually adding a 1 to the type of x, which of course is not ultimately === "number"
+
+
 **j)** `typeof x % 2 === "number"`
 
+Similarly, if x has been defined earlier as a number, this will output True.
+Console says: False. I assume for the same reasons that I was wrong on the last example. I think the next example should still be true, however, because it is evaluating the typeof everything in the ()'s rather than the first thing it encounters.
+
+
 **k)** `typeof (x % 2) === "number"`
```

Comment:
1k) Right! The interesting thing here is that `typeof` will always return a string, which when an operand for `%` will always return `NaN`, which is (weirdly enough) a number.